packages/block-library/src/social-link/block.json

```diff
@@ -34,6 +34,27 @@
 		"html": false,
 		"interactivity": {
 			"clientNavigation": true
+		},
+		"spacing": {
+			"margin": true,
```

Comment:
I'm not sure how useful allowing margin support on individual links is. My understanding of the social links block is that spacing should be handled via the parent Social Links block's block spacing support.